public/video-ui/src/pages/Video/index.jsx

```diff
@@ -245,7 +245,7 @@ class VideoDisplay extends React.Component {
       publishedVideo,
     } = this.props;
 
-    const { saving } = this.props.videoStates;
+    const { saving } = this.props.isSaving;
```

Comment:
This doesn't look correct either, `isSaving` shouldn't be an object. We could destructure `const { isSaving } = this.props`, or just reference `this.props.isSaving` where we are currently using `saving`.